nilearn/plotting/html_surface.py

```diff
@@ -80,7 +150,8 @@ def _check_mesh(mesh):
 
 def full_brain_info(volume_img, mesh='fsaverage5', threshold=None,
                     cmap=cm.cold_hot, black_bg=False, symmetric_cmap=True,
-                    vmax=None, vmin=None, vol_to_surf_kwargs={}):
+                    bg_map=None, bg_on_data=False, bg_map_rescale="auto",
```

Comment:
here the only values that can make sense for `bg_map` are `"auto"` and `None` -- shouldn't it be replaced with a Boolean parameter? and if not, can we add a comment and an `assert` to check that if it is not `"auto"` it is `None`

Reply:
Similarly to what happens in `view_img_on_surf()`, `bg_map` can actually be a numpy array here as well.
If we narrow down the possibilities to `"auto"` and `None`, I should probably get rid of `bg_map_rescale` altogether here.

In this case, considering the two options you propose, I'd be more in favour of making explicit assertions in the method about what values `bg_map` can take. In the long run, I think this will make it easier to make sure that `bg_map` is used consistently throughout the code base and to come up with a docstring definition that's common across methods.

Reply:
See my other comment -- I agree it is useful to use any bg map we want, I just don't see how it works for `view_img_on_surf` & friends (functions that take a volume image and plot both hemispheres) ATM, until SurfaceImage objects exist

Reply:
Yes, I agree it's a bit hazy.
Because the surface we project volumes to is always `fsaverage` - and users currently cannot tweak this - I expect users to give fsaverage background maps. I think it's nice that they can do this, although hard to understand at first. Maybe we should specify in the docstrings that the background map should be an `fsaverage` map?